### ISSUE TITLE
Docker creation fails

### ISSUE THREAD

**Issue by kilroy68** [OpenGeoscience/geonotebook#132]
I can no longer run the docker creation successfully.  I am attaching a screen shot of the error I am getting.
<img width="1094" alt="screen shot 2017-07-21 at 11 34 19 am" src="https://user-images.githubusercontent.com/647276/28473001-a3f4a968-6e08-11e7-98ec-80f0e9619b41.png">

**Comment by kilroy68** [comment]
So, I have determined a fix.  I edited the Dockerfile by adding pyOpenSSL to the pip command.

```
RUN pip2.7 install -U jupyter notebook \
                   mapnik \
                   pyproj \
                   ipywidgets \
                   scikit-image \
                   pyOpenSSL
```

**Comment by kotfic** [comment]
@kilroy68 awesome,  i was actually going to look into this right now.  I will test locally and get a PR in today

**Comment by kotfic** [comment]
@kilroy68  This should be closed by #133.  Thanks for the fix!
